docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -190,6 +190,7 @@ guaranteed because plugin developers can override this functionality. The follow
   <tr>
     <td>
       <ul>
+        <li>Dynatrace</li>
```

Comment:
what about this one?

Reply:
Dynatrace was updated on main in [PR 97744](https://github.com/grafana/grafana/pull/97744), but not in these older docs, so I figured I'd fix both things at once.